web/src/main/java/org/fao/geonet/monitor/onlineresource/CheckInfo.java

```diff
@@ -1,11 +1,29 @@
 package org.fao.geonet.monitor.onlineresource;
 
 public class CheckInfo {
-    public boolean status;
-    public long timestamp;
+
+    private CheckResult checkResult;
+    private long timestamp;
```

Comment:
Need to track down all previous references and change to the getter - I get the following compile error now.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:compile (default-compile) on project geonetwork-main: Compilation failure
[ERROR] /home/craigj/git-repos/aodn/core-geonetwork/web/src/main/java/org/fao/geonet/monitor/onlineresource/OnlineResourceInfo.java:[43,63] error: timestamp has private access in CheckInfo
[ERROR] -> [Help 1]
```